platform_plugin_turnitin/turnitin.py

```diff
@@ -96,10 +97,14 @@ def get_user_data(self):
         """
         user_service = self.runtime.service(self, "user")
         current_user = user_service.get_current_user()
+        user_full_name = current_user.full_name.split()
         return {
             "user_id": current_user.opt_attrs["edx-platform.user_id"],
             "user_email": current_user.emails[0],
-            "user_name": current_user.full_name.split(),
+            "user_name": user_full_name[0] if user_full_name else "no_name",
+            "user_last_name": " ".join(user_full_name[1:])
+            if len(user_full_name) > 1
+            else "no_last_name",
```

Comment:
I don't think you need to prefix all with `user` since get_user_data is pretty explicit.
```suggestion
            "name": full_name[0] if full_name else "",
            "lastname": full_name[1] if full_name[1:] else "",
```